components/formats-bsd/test/loci/formats/utests/GenericExcitationMapTest.java

```diff
@@ -132,20 +136,21 @@ public void testGenericExcitationSourceMapContent() throws Exception {
 
     assertNotNull(ome.getInstrument(0).getLightSource(0));
     GenericExcitationSource geSource = (GenericExcitationSource) ome.getInstrument(0).getLightSource(0); 
-    Map dataMap = geSource.getMap();
+    LinkedListMultimap<String,String> dataMap = geSource.getMap();
 
-    assertEquals(5, dataMap.getPairs().size());
+    assertEquals(6, dataMap.size());
     assertPair(dataMap, 0, "a", "1");
-    assertPair(dataMap, 1, "b", "2");
+    assertPair(dataMap, 1, "d", "2");
     assertPair(dataMap, 2, "c", "3");
-    assertPair(dataMap, 3, "d", "4");
+    assertPair(dataMap, 3, "b", "4");
     assertPair(dataMap, 4, "e", "5");
+    assertPair(dataMap, 5, "c", "6");
 
   }
 
-  void assertPair(Map dataMap, int idx, String name, String value) {
-    assertEquals(name, dataMap.getPairs().get(idx).getName());
-    assertEquals(value, dataMap.getPairs().get(idx).getValue());
+  void assertPair(LinkedListMultimap dataMap, int idx, String name, String value) {
```

Comment:
Make `LinkedListMultimap` a `LinkedListMultimap<String, String>` and you won't need the casts.